lib/logstash/inputs/lumberjack/server.rb

```diff
@@ -0,0 +1,29 @@
+# encoding: utf-8
+require "lumberjack/server"
+
+module Lumberjack
+
+  class LumberjackConnection < Connection
+    def initialize(fd, codec)
+      super(fd)
+      @codec = codec
+    end
+
+    def data(sequence, map, &block)
+      block.call(map, @codec)
+      super(sequence, map)
+    end
+  end
+
+
+  class Server
+    def run(codec, &block)
```

Comment:
I don't think we need to override Lumberjack::Server, I'll comment elsehere to show why